src/elm/Resolver/Decoder.elm

```diff
@@ -9,9 +9,19 @@ import TimeDuration.Model exposing (..)
 
 statsAndErrorsDecoder : Decoder ( Stats, Errors )
 statsAndErrorsDecoder =
+    oneOf [ notStartedDecoder, existingStatsAndErrorsDecoder ]
+
+
+existingStatsAndErrorsDecoder : Decoder ( Stats, Errors )
+existingStatsAndErrorsDecoder =
     map2 (,) statsDecoder errors
 
 
+notStartedDecoder : Decoder ( Stats, Errors )
+notStartedDecoder =
+    map2 (,) (null NotStarted) (succeed Nothing)
```

Comment:
Is the `Errors` type an alias for `Maybe` ? 😖

Reply:
This decoder confuses me. It's two constants?

Reply:
Yes, `type alias Errors = Maybe (List Error)`. I decided not to tackle the modeling of that here.

This decoder is used when the response returns `null`, and yes, it's a tuple of stats (`NotStarted`, which is a nullary data constructor) and `Nothing` (since there are no errors).